contracts/src/libraries/OmniPredeploys.sol

```diff
@@ -7,5 +7,5 @@ pragma solidity ^0.8.12;
  */
 library OmniPredeploys {
     // TODO: replace with actual predeploy address
-    address public constant OMNI_ETH_RESTAKING = address(0);
+    address public constant ETH_STAKE_INBOX = address(12_345);
```

Comment:
maybe move this to `test`, so that it's clear we're not inputting a dummy address into part of the protocol